src/background/modules/AutoRemoteFollow.js

```diff
@@ -144,6 +144,16 @@ function getInteractionType(url) {
     return [null, null];
 }
 
+/**
+ * Handles changes to the URL of a tab.
+ * @returns {void}
```

Comment:
```suggestion
 *
 * @returns {void}
```

Reply:
I want this feature. So I made changes according to my understanding of the requests ... https://github.com/hueyy/mastodon-simplified-federation/pull/1 

@hueyy **if you choose to accept this PR**, you may Ping @rugk to accept yours, and save yourself the minor time of making the changes yourself.

Reply:
Thanks! Let's merge this.